recipes/opengv/all/conanfile.py

```diff
@@ -54,26 +48,14 @@ def validate(self):
         #FIXME disable this one CCI has more RAM available
         if self.settings.compiler == "gcc" and self.options.shared:
             raise ConanInvalidConfiguration("Shared builds not supported with gcc since CCI errors out due to excessive memory usage.")
+        if self.settings.compiler == "clang" and self.settings.compiler.version == 12:
+            raise ConanInvalidConfiguration("Clang 12 builds fail on Conan CI.")
+
+    def layout(self):
+        cmake_layout(self)
 
     def source(self):
-        tools.get(**self.conan_data["sources"][self.version],
-                  destination=self._source_subfolder, strip_root=True)
-
-    def _configure_cmake(self):
-        if self._cmake:
-            return self._cmake
-        self._cmake = CMake(self)
-        self._cmake.definitions["BUILD_TESTS"] = False
-        self._cmake.definitions["BUILD_PYTHON"] = self.options.with_python_bindings
-        self._cmake.definitions["BUILD_POSITION_INDEPENDENT_CODE"] = self.settings.os != "Windows" and self.options.get_safe("fPIC", True)
-        if tools.cross_building(self):
-            cmake_system_processor = {
-                "armv8": "aarch64",
-                "armv8.3": "aarch64",
-            }.get(str(self.settings.arch), str(self.settings.arch))
-            self._cmake.definitions["CONAN_OPENGV_SYSTEM_PROCESSOR"] = cmake_system_processor
-        self._cmake.configure()
-        return self._cmake
+        get(self, **self.conan_data["sources"][self.version], strip_root=True)
```

Comment:
```suggestion
        get(self, **self.conan_data["sources"][self.version], destination=self.source_folder, strip_root=True)
```
try to be explicit here. It's default value is `"."` but states to the source folder too, but explicit makes life easier to review